### PR TITLE
June 17 Poll Drafts

### PR DESCRIPTION
**Pull request by MakerJanSky**

### CODE REVIEW

File: governance/polls/SparkLend Ethereum - Update Morpho Vault Supply Caps - June 17, 2024.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: SparkLend Ethereum - Increase Morpho DAI Vault Supply Caps - June 17, 2024
+summary: Signal your support or opposition to increasing the DAI supply caps for select Morpho Spark DAI Vault on Ethereum mainnet.
+discussion_link: https://forum.makerdao.com/t/may-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/24327
```
Comment (0xldr) [inline governance/polls/SparkLend Ethereum - Update Morpho Vault Supply Caps - June 17, 2024.md:4]: This is the wrong link

```suggestion
discussion_link: https://forum.makerdao.com/t/jun-12-2024-proposed-changes-to-sparklend-for-upcoming-spell/24489
```

File: governance/polls/SparkLend Gnosis - Onboard USDC.e to SparkLend - June 17, 2024.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+---
+title: SparkLend Gnosis - Onboard USDC.e to SparkLend - June 17, 2024
+summary: Onboard USDC.e (Bridged Circle USDC) to Spark Protocol on Gnosis with included parameters.
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard USDC.e to SparkLend - June 17, 2024.md:3]:
```suggestion
summary: Signal your support or opposition to onboarding USDC.e (Bridged Circle USDC) to Spark Protocol on Gnosis with included parameters.
```

File: governance/polls/SparkLend Gnosis - Onboard USDC.e to SparkLend - June 17, 2024.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+This poll allows the MakerDAO governance community to signal their support or opposition to adding USDC.e (Bridged Circle USDC) to Spark Protocol on Gnosis with the parameters below:
+
+### Spark Parameters
+
+* Token: USDC.e
+* Token Address: [0x2a22f9c3b484c3629090FeED35F17Ff8F88f76F0](https://gnosisscan.io/token/0x2a22f9c3b484c3629090feed35f17ff8f88f76f0)
+* Oracle: 1:1 USD Fixed
+* Oracle Provider: N/A
+* Borrow Enabled: Yes
+* Flash Loan Enabled	Yes
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard USDC.e to SparkLend - June 17, 2024.md:38]:
```suggestion
* Flash Loan Enabled: Yes
```

File: governance/polls/SparkLend Gnosis - Onboard USDC.e to SparkLend - June 17, 2024.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+* Stable Rate Borrowing: No
+* Collateral Enabled: No
+* [Isolation Mode](https://docs.spark.fi/defi-infrastructure/sparklend/spark-lend-features#isolation-mode) Enabled: No
+* Isolation Mode Debt Ceiling: n/a
+* Loan To Value: 0%
+* Liquidation Threshold: 0%
+* Liquidation Bonus: 0%
+* Liquidation Protocol Fee:	0%
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard USDC.e to SparkLend - June 17, 2024.md:46]:
```suggestion
* Liquidation Protocol Fee: 0%
```

File: governance/polls/SparkLend Gnosis - Onboard USDC.e to SparkLend - June 17, 2024.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+* Base borrow rate: 0%
+* Slope1: 9%
+* Slope2:	15%
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard USDC.e to SparkLend - June 17, 2024.md:49]:
```suggestion
* Slope2: 15%
```

**Review by 0xldr — CHANGES_REQUESTED** [review]
Please see our comments

File: governance/polls/SparkLend Gnosis - Onboard USDC.e to SparkLend - June 17, 2024.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+* Slope2: 15%
+* Optimal Utilization: 95%
+* Reserve factor: 10%
+* Borrowable in isolation mode: Yes
+* [Siloed borrowing](https://docs.spark.fi/defi-infrastructure/sparklend#siloed-borrowing): No
+* [Efficency Mode](https://docs.spark.fi/defi-infrastructure/sparklend#efficiency-mode-emode) "Emode" Category:	0
+* [Isolation Mode](https://docs.sparkprotocol.io/developers/features/isolation-mode) Enabled: N/A
+* Isolation Mode Debt Ceiling: N/A
+* [Supply cap](https://docs.spark.fi/defi-infrastructure/sparklend#supply-and-borrow-caps): 10,000,000 USDC.e
+* [Borrow cap](https://docs.spark.fi/defi-infrastructure/sparklend#supply-and-borrow-caps):  8,000,000 USDC.e
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard USDC.e to SparkLend - June 17, 2024.md:58]:
```suggestion
* Token: **USDC.e**
* Token Address: **[0x2a22f9c3b484c3629090FeED35F17Ff8F88f76F0](https://gnosisscan.io/token/0x2a22f9c3b484c3629090feed35f17ff8f88f76f0)**
* Oracle: 1:1 USD Fixed
* Oracle Provider: N/A
* Borrow Enabled: **Yes**
* Flash Loan Enabled: **Yes**
* Stable Rate Borrowing: **No**
* Collateral Enabled: **No**
* [Isolation Mode](https://docs.spark.fi/defi-infrastructure/sparklend/spark-lend-features#isolation-mode) Enabled: **No**
* Isolation Mode Debt Ceiling: n/a
* Loan To Value: **0%**
* Liquidation Threshold: **0%**
* Liquidation Bonus: **0%**
* Liquidation Protocol Fee: **0%**
* Base borrow rate: **0%**
* Slope1: **9%**
* Slope2: **15%**
* Optimal Utilization: **95%**
* Reserve factor: **10%**
* Borrowable in isolation mode: **Yes**
* [Siloed borrowing](https://docs.spark.fi/defi-infrastructure/sparklend#siloed-borrowing): **No**
* [Efficency Mode](https://docs.spark.fi/defi-infrastructure/sparklend#efficiency-mode-emode) "Emode" Category:	**0**
* [Isolation Mode](https://docs.sparkprotocol.io/developers/features/isolation-mode) Enabled: N/A
* Isolation Mode Debt Ceiling: N/A
* [Supply cap](https://docs.spark.fi/defi-infrastructure/sparklend#supply-and-borrow-caps): **10,000,000 USDC.e**
* [Borrow cap](https://docs.spark.fi/defi-infrastructure/sparklend#supply-and-borrow-caps):  **8,000,000 USDC.e**
```

File: governance/polls/SparkLend Ethereum - Update Morpho Vault Supply Caps - June 17, 2024.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes to the [Morpho Spark DAI vault](https://morpho.blockanalitica.com/metamorpho/vaults/0x73e65dbd630f90604062f6e02fab9138e713edd9):
+
+- Increase the supply cap of the [sUSDe 91.5% LLTV pool](https://morpho.blockanalitica.com/markets/1247f1c237eceae0602eab1470a5061a6dd8f734ba88c7cdc5d6109fb0026b28/) by 100 million DAI from 100 million DAI to **200 million DAI**.
+- Increase the supply cap of the [sUSDe 86% LLTV pool](https://morpho.blockanalitica.com/markets/39d11026eae1c6ec02aa4c0910778664089cdd97c3fd23f68f7cd05e2e95af48/) by 100 million DAI from 400 million DAI to **500 million DAI**.
+
+
```
Comment (0xldr) [inline governance/polls/SparkLend Ethereum - Update Morpho Vault Supply Caps - June 17, 2024.md:34]:
```suggestion
```

**Review by 0xldr — CHANGES_REQUESTED** [review]

File: governance/polls/SparkLend Ethereum - Update Morpho Vault Supply Caps - June 17, 2024.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
+---
+title: SparkLend Ethereum - Increase Morpho DAI Vault Supply Caps - June 17, 2024
```
Comment (VoteWizard) [inline governance/polls/SparkLend Ethereum - Update Morpho Vault Supply Caps - June 17, 2024.md:2]:
```suggestion
title: SparkLend Ethereum - Update Morpho DAI Vault Supply Caps - June 17, 2024
```

Shouldn't this title match the file name and the title below?

Comment (0xldr) [reply governance/polls/SparkLend Ethereum - Update Morpho Vault Supply Caps - June 17, 2024.md:2]: yes, we will update this

Comment (0xldr) [reply governance/polls/SparkLend Ethereum - Update Morpho Vault Supply Caps - June 17, 2024.md:2]: https://github.com/makerdao/community/pull/2671

**Pull request merged**